apps/src/netsim/netsim.js

```diff
@@ -964,11 +964,16 @@ function resizeFooterToFitToLeftOfContent() {
     return;
   }
 
+  var padding = parseInt(window.getComputedStyle(smallFooter)["padding-left"]);
+
+  var boundingWidth;
   if (leftColumn && $(leftColumn).is(':visible')) {
-    smallFooter.style.maxWidth = leftColumn.getBoundingClientRect().right + 'px';
+    boundingWidth = leftColumn.getBoundingClientRect().right;
   } else if (instructions && $(instructions).is(':visible')) {
-    smallFooter.style.maxWidth = instructions.getBoundingClientRect().right + 'px';
+    boundingWidth = instructions.getBoundingClientRect().right;
   }
+
+  smallFooter.style.maxWidth = (boundingWidth) ? (boundingWidth - padding) + 'px' : null;
```

Comment:
I completely agree. In particular, having netsim directly modify the small footer when they should be relatively independent entities feels dirty.

Thanks!